### ISSUE TITLE
GPS tags on mobile devices

### ISSUE THREAD

**Issue by gonzika** [MikeKovarik/exifr#64]
Hello! I have problem with parse GPS tags on mobile devices. On destop all working well.
![image](https://user-images.githubusercontent.com/63928278/120917334-632adb80-c6b7-11eb-974f-4b777acbc448.png)

But with same photo, on mobile I can not parse no one GPS tags. 
![image](https://user-images.githubusercontent.com/63928278/120917380-9ff6d280-c6b7-11eb-9491-a95fc04c7eb9.png)

How can i get this tags? Here https://mutiny.cz/exifr/ all working well.
Thank you

**Comment by MikeKovarik** [comment]
Hello. Can you describe the problem a little more?
Is it the same code/app, same photo, same actions, but different result on phone?
Could you share the photo, or the photo? Or anything that could help me find out what's wrong.
Thanks

**Comment by gonzika** [comment]
Yes, I can! While i making test projects, I had a suspension on my React.
Firstly what I do, its vanilla html projects and he works well on all platforms. https://github.com/gonzika/test (photo for testing with gps tags here)
Then I trying to integrate different ways in my react-projects, i including your library from CDN (normaly it installed from npm), i even turning off all rendering, but it does not help.

Test projects where I can retry bug here https://codepen.io/gonzika/pen/QWprXdK
You can see, two different ways (i thinked what its babel's guilt, but libray from CDN and logic in index's <script> does not work too.

**Comment by bdk0172** [comment]
Having a similar issue in a React project where I'm using Capacitor Camera plugin in conjunction with exifr.  I've got it set up to create an Android project and to verify that exifr was working I have 2 alerts set up, one to display make and model and another to display lat/lon.  When I run it directly from VS Code as a PWA I can attach an image that I've transferred from my Android phone and it will display the correct lat/lon in the alert.  When I run it on my phone through Android Studio and take a picture, I get the make and model alert, but not the lat/long alert I'm assuming because it's getting an error.  When I attach the same photo that gave me the correct lat/lon in PWA, it gives me "0/0" when running on my phone.  I'm guessing there's a location permissions issue that needs to be set in the Androidmanifest.xml, but I haven't figured that out yet.

**Comment by 19th** [comment]
I have a similar issue, but the problem is not in the library, but in the device. Android and iOS strips down geolocation data on file upload. On Android - if I choose to browse files with another app, then geolocation data is preserved.

**Comment by Ivwc** [comment]
you can try input without accept.
on Android is work

**Comment by josephfrazier** [comment]
I'm experiencing a similar issue, and I think I've been able to rule out Android metadata-stripping as the cause. I may try to put together a minimal test case to make debugging easier, but I've starting describing my problem and work-in-progress to fix it at https://github.com/josephfrazier/Reported-Web/pull/360, including a bit where I'm md5sum'ing the image uploaded by the browser, and getting the same result on both desktop and mobile.

**Comment by josephfrazier** [comment]
I was just revisiting this issue, and wanted to confirm for anyone else who ends up here that **removing the `accept` attribute from the `<input>` element worked for me**, as @Ivwc said above. It was due to Android stripping metadata after all, and my previous comment was mistaken (I encountered [this bug](https://github.com/pvorb/node-md5/issues/49) in the md5 sum package I was using)
